source/customization.rst

```diff
@@ -244,6 +252,27 @@ In ``/etc/ood/config/apps/shell/env`` set the env var ``DEFAULT_SSHHOST`` to cha
 
 This will control what host the shell app ssh's to when the URL accessed is ``/pun/sys/shell/ssh/default`` which is the URL other apps will use (unless there is context to specify the cluster to ssh to).
 
+SSH Wrapper
```

Comment:
Rename "Shell App SSH Command Wrapper" and in the first sentence clarify this is for the shell app.